frontend/safdarjung-tomb-explorer/style.css

```
@@ -0,0 +1,514 @@
+  --saf-accent-strong: #047857;
+  --saf-gold: #a16207;
+  --saf-heading: #1a241f;
+  --saf-overlay: rgba(247, 245, 239, 0.82);
+  --saf-shadow: 0 12px 28px rgba(36, 48, 42, 0.14);
+}
+
+/* ---------------- Base ---------------- */
+.safdarjung-main {
+  background: var(--saf-bg);
+  color: var(--saf-text);
+  font-family: "Outfit", "Segoe UI", system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)